client/src/redux/actions/related.js

```diff
@@ -1,6 +1,5 @@
-const setRelatedProduct = (product) => ({
-  type: 'SET_RELATED_PRODUCT',
+export const SET_RELATED_PRODUCT = 'SET_RELATED_PRODUCT';
```

Comment:
Why is this being exported?

Reply:
Because if I set it to a variable VS code will help auto fill instead of typing out all the "type".

Reply:
When I am writing code in my components

Reply:
not sure I follow - you don't actually refer to "SET_RELATED_PRODUCT" elsewhere in your code do you?

Reply:
Not yet, but I will use it in my components when I refer to my reducer later.